OpenEdXMobile/res/values/strings_2.xml

```diff
@@ -8,4 +8,6 @@
     <string name="label_leaving_the_app">Leaving the app</string>
     <!-- Dialog Alert Message for Open in browser -->
     <string name="leaving_the_app_message">You are now leaving the {platform_name} app and opening a browser.</string>
+    <!-- Toggle to allow downloads using cellular data -->
+    <string name="label_accept">Accept</string>
```

Comment:
IMO, we should move it to `iap_strings` as well assuming they might only order translations for IAP strings. 
What do you think? 

And I think the comment is not relevant as well. It could be something like i.e 'Alert dialog accept button`

Reply:
comment updated to `Alert dialog accept button label`, but retain the string in the `strings_2.xml` as it seems to be general.